e2e/index.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { expect, test } from './fixtures'
+import { containerID, popupThumbID, popupCardID } from '../src/browser-extension/content_script/consts'
+
+test('popup card should be visible', async ({ page }) => {
+    await page.goto('https://example.com')
```

Comment:
I don't think we should use this domain name. It's an inaccessible address that will cause continuous loading.